koku/api/report/azure/openshift/provider_map.py

```diff
@@ -38,7 +38,7 @@ def __init__(self, provider, report_type):
                 "provider": Provider.OCP_AZURE,
                 "alias": "subscription_guid",
                 "annotations": {"cluster": "cluster_id"},
-                "end_date": "costentrybill__billing_period_end",
+                "end_date": "costentrybill__billing_period_start",
```

Comment:
I don't quite understand why we aren't changing this one to `usage_start` as well. From what I understand, we are changing these to fields on which we index, but this is not an indexed field...is it?

Reply:
Yes. It is part of a multi-column index. It is the first entry in the index tuple, so I think this will work.